packages/perseus/src/widgets/__testutils__/test-data-util.ts

```diff
@@ -0,0 +1,32 @@
+import type {PerseusItem, PerseusRenderer} from "@khanacademy/perseus-core";
+
+/**
+ * Creates a Perseus item with the provided question/PerseusRenderer data.
+ * Useful for converting test data used with RendererWithDebugUI to test data
+ * for ServerItemRendererWithDebugUI.
+ *
+ * @param {PerseusRenderer} question - The question to be included in the Perseus item.
+ * @returns {PerseusItem} The created Perseus item.
+ */
```

Comment:
Regarding docs.  For things that are self-evident, I prefer to avoid adding a doc comment.  So I think I'd just recommend this (as the `question` param is quite obvious as is the return value): 

```suggestion
/**
 * Creates a Perseus item with the provided question/PerseusRenderer data.
 * Useful for converting test data used with RendererWithDebugUI to test data
 * for ServerItemRendererWithDebugUI.
 */
```

Reply:
Thanks! Ended up removing this file, but glad to know my thought to remove the parameter definitions was correct.